SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -2423,6 +2423,18 @@ void USpatialReceiver::ResolveIncomingOperations(UObject* Object, const FUnrealO
 			}
 		}
 
+		ObjectsToInspect.Add(ChannelObjectsToBeResolved{ DependentChannel, ReplicatingObject, RepState });
+	}
+
+	for (const auto& Objects : ObjectsToInspect)
+	{
+		USpatialActorChannel* DependentChannel = Objects.Channel;
+		if (!Objects.Object.IsValid())
+		{
+			continue;
+		}
+		UObject* ReplicatingObject = Objects.Object.Get();
```

Comment:
Not sure how Unreal weak ptrs work, maybe this isn't needed 
```suggestion
		UObject* ReplicatingObject = Objects.Object.Pin();
```

Reply:
Looks like you can't pin TWeakObjectPtrs, which I guess makes sense.
UObjects are garbage collected by Unreal when they are destroyed, so pinning to extend their lifetime would be weird.
Also, the object can't be destroyed until the repnotifies happen, and I am in the game thread, so I think I am safe.
If it's destroyed by the repNotify, I don't use this pointer after that point.